packages/select-rich/test/lion-select-rich.test.js

```diff
@@ -254,10 +254,14 @@ describe('lion-select-rich', () => {
       ));
 
       expect(el.opened).to.be.true;
-      // a click on the button will trigger hide on outside click
+      // a mousedown (a full click won't work) on the button will trigger hide on outside click
       // which we then need to sync back to "opened"
       outerEl.click();
```

Comment:
Nitpick:

Feel free to remove the .click, I don't think we have to verify in the test that it doesn't work :P more important to test what does work (mousedown), but it's up to you as well, I don't mind keeping it.